src/ContainerLogs.jsx

```diff
@@ -95,47 +95,41 @@ class ContainerLogs extends React.Component {
         }
         this.resize(this.props.width);
 
-        const logsData = {};
-        logsData.name = this.props.containerId;
-
-        varlink.connect(utils.getAddress(this.props.system), this.props.system)
-                .then(connection => {
-                    connection.monitor("io.podman.GetContainerLogs", logsData, this.onStreamMessage)
-                            .then(this.onStreamClose)
-                            .catch(e => {
-                                if (e.error === "ConnectionClosed")
-                                    this.onStreamClose();
-                                else
-                                    this.setState({
-                                        errorMessage: e.message,
-                                        streamer: null,
-                                    });
-                            });
-                    this.setState({
-                        streamer: connection,
-                        errorMessage: "",
-                    });
-                })
+        const connection = rest.connect(client.getAddress(this.props.system), this.props.system);
+        const options = {
+            method: "GET",
+            path: "/v1.12/libpod/containers/" + this.props.containerId + "/logs",
+            body: "",
+            params: {
+                follow: true,
+                stdout: true,
+                stderr: true,
+            },
+        };
+
+        connection.monitor(options, this.onStreamMessage, this.props.system, true)
+                .then(this.onStreamClose)
                 .catch(e => {
                     this.setState({
                         errorMessage: e.message,
                         streamer: null,
                     });
                 });
+        this.setState({
+            streamer: connection,
+            errorMessage: "",
+        });
     }
 
     onStreamMessage(data) {
-        // data = {parameters: {container: [<data_string>]}}
-        // Example <data_string>="2020-01-23T09:34:34.563845178+01:00 stdout F 19\r\n"
-        if (data && data.parameters) {
-            const just_logs = data.parameters.container.map(e => e.split(" ").slice(3)
-                    .join(" "));
+        if (data) {
             if (this.state.loading) {
                 this.state.view.reset();
                 this.state.view._core.cursorHidden = true;
                 this.setState({ loading: false });
             }
-            this.state.view.write(just_logs.join(""));
+            // HACK: Every line starts with `<0d><01><00><00><00><00><00><00>` so remote this garbage
```

Comment:
remo"v"e. No bug report then? If it doesn't make sense and just gets in the way, it should be fixed in the API IMHO. Also, what happens if one of these days the prefix gets dropped? We'd then lose the first 8 bytes of output.

Reply:
Ok, so I found in (docker) docs what it means. Linked to that. I believe that we don't actually need to read the amount specified in the header, since from what I saw it was 0, so maybe podman does not implement that, just keeps the same format.

Reply:
Yay, thanks for figuring this out! That's a pretty badly broken header :-)